mix.exs

```diff
@@ -83,10 +83,10 @@ defmodule Unifex.MixProject do
 
   defp deps do
     [
+      {:membrane_core, "~> 0.10.2"},
```

Comment:
Do we need it there? Unifex shouldn't have dependency to core

Reply:
I cannot use our logger without it

Reply:
use https://hexdocs.pm/logger/1.12.3/Logger.html